src/quick_error.rs

```diff
@@ -331,7 +331,9 @@ macro_rules! quick_error {
     (FIND_DISPLAY_IMPL $name:ident $item:ident: $imode:tt
         { display($self_:tt) -> ($( $exprs:tt )*) $( $tail:tt )*}
     ) => {
-        |quick_error!(IDENT $self_): &$name, f: &mut ::std::fmt::Formatter| { write!(f, $( $exprs )*) }
```

Comment:
This is the line that was causing problems. Rustfmt will complain if you have a line longer than 100 characters.